next.config.ts

```diff
@@ -53,7 +52,17 @@ const nextConfig: NextConfig = {
         headers: [
           {
             key: "Content-Security-Policy",
-            value: getContentSecurityPolicy(),
+            value: [
+              "default-src 'self'",
+              "script-src 'self' 'unsafe-inline' 'unsafe-eval' https://stats.data.gouv.fr/",
+              "style-src 'self' 'unsafe-inline'",
+              "img-src 'self' data:",
+              "frame-src 'self' https://stats.data.gouv.fr/ https://plugins.crisp.chat/",
+              "connect-src 'self' https://stats.data.gouv.fr/ https://errors.data.gouv.fr/ https://bodacc-datadila.opendatasoft.com/ https://data.economie.gouv.fr/ https://journal-officiel-datadila.opendatasoft.com/ https://api-lannuaire.service-public.fr/ https://data.culture.gouv.fr/ https://data.inpi.fr/ https://openmaptiles.geo.data.gouv.fr/ https://openmaptiles.data.gouv.fr/ https://geo.api.gouv.fr https://api-adresse.data.gouv.fr https://tabular-api.data.gouv.fr https://koumoul.com",
```

Comment:
This limits the endpoints from which we can use `fetch`, right ? 👀 (but only from the client since it's a response header)